src/libraries/System.Private.CoreLib/src/System/Resources/ManifestBasedResourceGroveler.cs

```diff
@@ -321,43 +321,36 @@ private static ResourceSet InternalGetResourceSetFromSerializedData(Stream store
             return rs;
         }
 
-        private Stream? GetManifestResourceStream(Assembly satellite, string fileName)
+        private static Stream? GetManifestResourceStream(Assembly satellite, string fileName)
```

Comment:
Same here, not sure how this relates to the BaseName change.

Reply:
Previously, this couldn't be static because it accessed `_mediator`, but that's no longer true since `fileName` now contains the full information in all cases. With that change, I believe I was getting an analyzer error until I made this static.